app/components/TextArea/index.js

```diff
@@ -1,32 +1,23 @@
+// @flow
 import React from 'react'
-import PropTypes from 'prop-types'
 
-class TextArea extends React.Component {
-  constructor(props) {
-    super(props)
-    this.onChange = this.onChange.bind(this)
-  }
-
-  onChange(...args) {
-    this.props.onChange(...args)
-  }
+type Props = {
+  name?: string,
+  value?: string,
+  onChange?: Function,
```

Comment:
Can we be more explicit about the parameters for the function and its return value?